### PR TITLE
Added 'q' unit to Sass

### PR DESCRIPTION
**Pull request by royels**
Addressing issue #1753.

### CODE REVIEW

File: doc-src/SASS_CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 * Table of contents
 {:toc}
 
+## 4.0.0 (Unreleased)
+
+* Combine ids and `:root` when unifying selectors with `@extend` and selector
+  functions.
+
+* Add a `$weight` parameter to `invert()`.
+
```
Comment (chriseppstein) [inline doc-src/SASS_CHANGELOG.md:13]: I'm confused why this is part of the patch.

**Comment by chriseppstein** [comment]
Patch looks great, but it seems like some of the changelog from master branch ended up in this pull request. Let's get that cleaned up and we'll merge it in.

**Comment by royels** [comment]
Sorry I didn't respond sooner! Cleaned out the changelog text that sneaked into the PR.

**Comment by royels** [comment]
Resolved merge conflicts from updating to 3.4.19.

**Comment by royels** [comment]
Resolved conflicts from the change to 3.4.19.

**Pull request closed without merging**